Lesson 03/translator_client/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>A simple Translator</title>
+	<link rel="stylesheet" href="styles/main.css">
+</head>
+<body>
+	<div class="formWrapper">
+		<h2 class="title">A simple translator</h2>
+		<form action="#" id="translateForm">
+			<fieldset>
+				<label for="serverPort">Input server port:</label>
+				<input type="number" name="serverPort" id="serverPort" value="3000" min="1024" max="65535" maxlength="5">
+				&nbsp;
+				<span class="hint">(1024 - 65535)</span>
+				<br><br>
+				<label for="lang">Select translation direction:</label>
+				<select name="lang" id="lang">
+					<option value="ru-en">Russian &rarr; English</option>
+					<option value="en-ru">English &rarr; Russian</option>
+				</select>
+				<br><br>
+				<label for="text">Type some text to translate:</label>
+				<div class="textWrapper">
+					<textarea name="textSource" id="textSource" rows="10" placeholder="Привет!"></textarea>
+					<span class="translate">&darr;</span>
+					<textarea name="textResult" id="textResult" rows="10" placeholder="Hello!" readonly></textarea>
+				</div>
+				<div class="buttonsWrapper">
+					<input class="button" type="submit" value="Translate">
+					&nbsp;
+					<input class="button" type="reset" value="Reset">
+				</div>
+			</fieldset>
+		</form>
+	</div>
+	<script src="packages/jquery/jquery-3.3.1.min.js"></script>
```

Comment:
It could be installed via npm as well

Reply:
You should install it as usual in a root directory and then insert a script tag from the node_modules folder. Something like following:
```html
<script src="node_modules/jquery/dist/jquery.min.js"></script>
```
BTW the best way is using a module bundler like webpack, but it's out of the scope of our course.

Reply:
Ok, I'll look into webpack myself then. It seems to me, that installing and linking **jQ** to a web page with **npm** - not the best way, though it works quite fine.

Reply:
Actually, if u don't like this approach u could try to serve vendor libraries by express.static. It could help with a custom path.